core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -928,11 +826,11 @@ private object RTS {
                 try cancel()
                 catch {
                   case t: Throwable if (rts.nonFatal(t)) =>
-                    fork(unhandled(t :: Nil), unhandled)
+                    fork(IO.terminate(t), unhandled)
```

Comment:
This is a change from the old version. I _think_ it will have the same effect. /cc @regiskuckaertz

Reply:
Hmm, I don't think so. This will evaluate in `IO.Tags.Fail` with no finalizer, meaning it will just produce `ExitResult.Failed(Cause.unchecked(t))`, whereas `unhandled(Cause.unchecked(t))` will evaluate the supervision.

Reply:
Ooh, but the result is evaluated in `reportErrors`! So yes, that;s the same 😄

Reply:
Maybe we should revert just in case. 😄